vm/compiler/src/ledger/block/genesis.rs

```diff
@@ -34,8 +34,10 @@ impl<N: Network> Block<N> {
         let inputs = [Value::from_str(&caller.to_string())?, Value::from_str("1_100_000_000_000_000_u64")?];
         // Authorize the call to start.
         let authorization = vm.authorize(private_key, &program_id, function_name, &inputs, rng)?;
+        println!("authorized, genesis");
```

Comment:
```suggestion
```
won't clutter the PR with this suggestion anymore, but this and the other `println`s should probably be removed